Week03/lesson06/classwork/startup_matchmaker/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+
+<head>
+
+  <title>Startup Matchmaker</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,700,600italic" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Oswald:400,700,300" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather:400,300,700,700italic,900" rel="stylesheet" type="text/css">
+
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+
+<header>
+
+  <h1>Startup Matchmaker</h1>
+
+<nav>
+
+    <div class="pages">
+      <a href="#">Developers</a>
+      <a href="#">Designers</a>
+    </div>
+
+    <a href="#">How it Works</a>
+    <a href="#">Our Team</a>
+    <a href="#">Blog</a>
+
+</nav>
+
+  <p>Because two brains <span>taste</span> <em>are</em> better than one.</p>
+
+</header>
+
+<main>
+
+<div class="boxright">
+
+  <h2>Meet your Match!</h2>
+
+    <p>Have a great idea for a <br>
```

Comment:
Ideally you don't want to use `<br>`'s here, you want to somehow limit the width of the parent paragraph (which you've already done by limiting the width of `.boxright` to 400px) and then let the text flow within that space